packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2088,15 +2094,17 @@ export class LitNodeClientNodeJs extends LitCore {
           relayUrl: (params as ClaimRequest<'relay'>).relayUrl,
           relayApiKey: (params as ClaimRequest<'relay'>).relayApiKey,
         });
+      } else {
+        mintTx = await defaultMintClaimCallback({
+          derivedKeyId,
+          authMethodType: params.authMethod.authMethodType,
+          signatures: nodeSignatures,
+          pubkey,
+          relayUrl: (params as ClaimRequest<'relay'>).relayUrl,
```

Comment:
these changes were needed to support overriding the relayUrl and api key when no callback function is passed in